File: dotnet/cafdemalihapa/Files/FileService.cs
namespace cafdemalihapa.Files
{
    public static class FileService
    {
        public static void CreateNumberedFiles(string path)
        {
            var dashLine = new string('-', 40);
            var lines = Enumerable.Range(0, 20)
                .SelectMany<int, string>(index => index == 0
                    ? [dashLine]
                    : ["", "", "", dashLine])
                .ToList();

            foreach (var number in Enumerable.Range(1, 10))
            {
                var filePath = Path.Combine(path, number.ToString("D3"));
                File.WriteAllLines(filePath, lines);
            }
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Files/FileService.fs
namespace cafdemwimu.console.Files

open System
open System.IO

module FileService =
    let CreateNumberedFiles (path: string) =
        let dashLine = String('-', 40)
        let lines =
            seq { 0 .. 19 }
            |> Seq.collect (fun index ->
                if index = 0 then [ dashLine ]
                else [ ""; ""; ""; dashLine ])
            |> List.ofSeq

        for number in 1 .. 10 do
            let filePath = Path.Combine(path, number.ToString("D3"))
            File.WriteAllLines(filePath, lines)
